build.sbt

```
@@ -31,3 +33,23 @@ lazy val `dog-aging-hles-transformation` = project
   .settings(
     libraryDependencies += "com.beachape" %% "enumeratum" % enumeratumVersion
   )
+
+lazy val `dog-aging-orchestration-workflow` = project
```

I expect that we'll end up with multiple orchestration workflows in DAP for different data streams. I'm not sure of the best way to model it:
1. One top-level chart w/ separate workflow definitions (and toggles to enable/disable each)
2. A chart per top-level workflow

I'm leaning towards option 2, in which case I'd rename this:
```suggestion
lazy val `dog-aging-hles-orchestration` = project
```
And move it under the `hles` directory.

But I could be convinced of option 1 if others have strong opinions.

---

I agree that a chart per top-level workflow makes sense. 👍